src/memfd.rs

```diff
@@ -98,6 +98,7 @@ impl Default for MemfdOptions {
 }
 
 /// Page size for a hugetlb anonymous file.
+#[allow(clippy::all)]
```

Comment:
Not very fond of this, but on the other hand the new lint does not exist on older toolchains so it can't be directly whitelisted there.